dev/io.openliberty.org.apache.myfaces.4.0_fat/publish/servers/faces40_thirdPartyAPIServer/osgi.debug.options

```diff
@@ -0,0 +1,12 @@
+###############################################################################
```

Comment:
This file is not necessary for all of our servers, I kept it in the simpleTest server just as a reference in case we need it during dev.